### ISSUE TITLE
DSM 7.2 - Denverton DVA3221 does not accept lan card I219 (e1000) and I225 (igc).

### ISSUE THREAD

**Issue by vinhnguyen1104** [AuxXxilium/arc#207]
I have 3 lan cards: I219, I225, RTL8125... after upgrading the latest ARC version, I219, I225 doesn't work.

**Comment by AuxXxilium** [comment]
upgrade modules and try again

**Comment by AuxXxilium** [comment]
and are you sure you use .13?

**Comment by vinhnguyen1104** [comment]
> 

I use .12 . You just released .13, let me try again.

**Comment by AuxXxilium** [comment]
thats the problem with people they are not in my discord. the .12 build was only for testing and was deleted soon after build completed.

**Comment by vinhnguyen1104** [comment]
After updating the .13 version and updating the addon and module, the i219 and i225 still do not receive. The previous version I used from mid-April... i219 and i225 worked fine.

**Comment by AuxXxilium** [comment]
mid-arpil with 7.2?

**Comment by AuxXxilium** [comment]
do you have rebuilded the full loader and config?

**Comment by vinhnguyen1104** [comment]
I created a brand new bootable usb...
Oh I remember wrong, it was the end of April. Your first version supports 7.2RC.

**Comment by vinhnguyen1104** [comment]
I also tried wjz304's arpl-i18n v23.5.2 and got the same error...
I'm looking to retry your old modules and see if that works. But now I can't find the old img file anymore :(

**Comment by AuxXxilium** [comment]
please wait before doing anything. i upgrading modules atm

**Comment by AuxXxilium** [comment]
modules are building now: try update modules in loader after this job is done: https://github.com/AuxXxilium/arc-modules/actions/runs/4902660676

**Comment by AuxXxilium** [comment]
github build is buggy, please wait for my next response.

**Comment by AuxXxilium** [comment]
now you can try it

**Comment by vinhnguyen1104** [comment]
Thank you !
After updating to .13-newmodules I219 and I225 worked fine...

**Comment by vinhnguyen1104** [comment]
@AuxXxilium After I create a new bootable usb version .15 the I225 doesn't work...
RTL8125 and I219 work fine.

**Comment by AuxXxilium** [comment]
same modules than yesterday. nothing new.

**Comment by AuxXxilium** [comment]
you can try it with update modules. there is a later modules tree than arc release.

**Comment by AuxXxilium** [comment]
ok i see, i lost something. will fix it asap.

**Comment by vinhnguyen1104** [comment]
I manually updated the modules to 23.5.14-3 and it worked fine. When you release new modules I will update and report back to you the results...

**Comment by AuxXxilium** [comment]
sounds good. thank you

**Comment by AuxXxilium** [comment]
should work again with new modules tree. modules is ready to try.

**Comment by vinhnguyen1104** [comment]
I updated ARC .16 and arc-modules .22, I219 and I225 worked fine.
Thank...

**Comment by AuxXxilium** [comment]
Thank you for feedback. Will use this modules tree from now.
